uwblueprint/community-fridge-kw
# feat: emails when admin removes volunteer or delete checkin

## Brief description. What is this change? 


## Implementation description. How did you make this change?
* When admin removes volunteer from checkin or deletes a checkin, the volunteer should get an email



## Steps to test
1. Sign up for a fridge checkin shift as a volunteer. Make sure you get a confirmation email for checkin sign up. Admin should get one as well. If you want to test for admin, you can change the email in getAdmin() function locally to make sure admin is getting the email
2. Cancel your fridge checkin shift and make sure you get a cancellation email. Admin should also get a cancellation email
3. Sign in as admin. Remove the volunteer from the shift. The volunteer should get a cancellation email
4. As an admin, assign yourself as admin and make sure no one gets any email and there are no errors
5. As an admin, Remove yourself from volunteer shift, and make sure no one gets any email and there are no errors
6. As an admin, delete a checkin where a volunteer has signed up. A cancellation email should be sent to volunteer
7. As an admin, delete checkins by date range, and ensure that within that date range, you have checkins where volunteer has signed up and some where no volunteers are signed up. Emails should be sent to any checkin which had a volunteer signed up for it




## Checklist
